src/App.tsx

```diff
@@ -42,6 +43,7 @@ export default function App(): JSX.Element {
             <Route path="/friend-list" element={<FollowPage />} />
           </Route>
           <Route path="/withdrawal" element={<Withdrawal />} />
+          <Route path="profile" element={<MyPage />} />
```

Comment:
큰 차이는 없겠지만 다른 라우팅이랑 통일해두는게 좋아보여요
```suggestion
          <Route path="/profile" element={<MyPage />} />
```